bin/addons-linter

```diff
@@ -4,7 +4,13 @@ var path = require('path');
 
 var absoluteAppRoot = path.resolve(path.join(__dirname, '..'));
 global.appRoot = path.relative(process.cwd(), absoluteAppRoot);
-
-require('../dist/addons-linter')
-  .createInstance({runAsBinary: true})
-  .run();
+try{
+    require('../dist/addons-linter')
```

Comment:
I'm guessing, actually, that we don't lint this file. We should do that! I've filed #1487.